pages/integrations/slack.md

```diff
@@ -7,7 +7,21 @@ Configuring a Slack notification service will authorize access for your desired
 
 Setting up a Notification Service requires Buildkite organization admin access.
 
-## Adding a notification service
+## Adding notification services
+There are two ways to add notification services: workspace-wide and channel-specific. We reccomend using a workspace. Using a workspace only requires a single authorization for the entire workspace, allowing you to connect channels via YAML rather than multiple notifications services. The migration to a Workspace will not impact current integration implementation.
```

Comment:
```suggestion
There are two ways to add notification services: workspace-wide and channel-specific. We recommend using a workspace. Using a workspace only requires a single authorization for the entire workspace, allowing you to connect channels via YAML rather than multiple notifications services. The migration to a Workspace will not impact current integration implementation.
```